main.css

```diff
@@ -235,13 +252,127 @@ header {
 
 .contact-label {
   display: block;
-  margin-bottom: 0.5rem;
+  margin-bottom: 0.8rem;
 }
 
 .contact-field {
   width: 100%;
 }
 
 .contribute-message {
+  margin-bottom: 3.2rem;
+}
+
+.past-events {
+  position: relative;
+  padding-bottom: 6rem;
+  overflow: hidden;
+}
+
+.past-events .container {
+  position: relative;
+}
+
+.background-shapes {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  z-index: 0;
+}
+
+.pre-icon:before {
+  content:"";
+  display: inline-block;
+  vertical-align: middle;
+  width: 2.5rem;
+  height: 2.5rem;
+  background-size: contain;
+  background-position: center;
+  background-repeat: no-repeat;
+  margin-right: 1.6rem;
+}
+.event-time:before {
+  background-image: url(img/icon-calendar.svg);
+}
+.event-location:before {
+  background-image: url(img/icon-location.svg);
+}
+.event-register:before {
+  background-image: url(img/icon-register.svg);
+}
+.event-report:before {
+  background-image: url(img/icon-newspaper.svg);
+}
+
+.past-event + .past-event {
+  margin-top: 8rem;
+}
+
+.event-info {
+  flex-shrink: 0;
+  padding-right: 4rem;
   margin-bottom: 2rem;
 }
+.event-info a,
+.event-info time,
+.event-info p {
+  line-height: 2.4;
+  display: block;
+}
+
+.event-photos img {
+  width: 70%;
+  border: 4px solid #FF7657;
```

Comment:
Can we use HSL for consistency and maybe save this in a custom property (also for the other colors) ?